packages/core/types/src/http/workflow-execution/admin/entities.ts

```diff
@@ -1,11 +1,92 @@
+export type TransactionStepStatus =
```

Comment:
@carlos-r-l-rodrigues, perhaps you could glance over these and let me know if something is off.